CONTRIBUTING.md

```diff
@@ -4,6 +4,24 @@ This document should concisely express the project development status,
 methodology, and contribution process.  As the community makes progress, we
 should keep this document in sync with reality.
 
+## Submitting a Pull Request (PR)
+
+The following outlines the general rules we follow:
+
+- All PRs must have the appropriate documentation changes made within the
```

Comment:
Strongly disagree.  Veto-level disagree, for the reasons I've commented about here.  Wait until a CI run takes 2 hours -- you're not going to want to change a PR for doc.